updatedUsers.ts

```diff
@@ -52,10 +49,26 @@ const users: User[] =
             age: 20,
             physicalAddress: "Dubai"
         }
+    ];
+
+    function myProfile(userId: number): User | undefined {
```

Comment:
No need to pass a parameter of userId as we already told you that we need a user with id=103.(We already know a user we want).

Good thing: You create a reusable function to find any user by ID.(I like your creativity.) GOOD JOB